presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergConfig.java

```diff
@@ -52,4 +54,18 @@ public IcebergConfig setCompressionCodec(HiveCompressionCodec compressionCodec)
         this.compressionCodec = compressionCodec;
         return this;
     }
+
+    @NotNull
```

Comment:
Not needed since this returns a primitive

Reply:
removed, thanks